Junk4Dollars-appTests/Junk4Dollars_appTests.swift

```diff
@@ -7,7 +7,7 @@
 //
 
 import XCTest
-@testable import Junk4Dollars_app
+import Junk4Dollars_app
```

Comment:
What's this test class for?

Reply:
`@testable?` At least part of what it does is allow you to test any part of a class, even if it's not public. Eric talked about not using this, since you really shouldn't be testing private methods, and it can sometimes allow you to think something is public (since you're testing it), even when it's not - leading to problems down the road in development.

Reply:
Sorry, I meant the whole test file. Doesn't seem to be testing anything.